deployment/terraform/container-service.tf

```diff
@@ -45,54 +21,40 @@ resource "aws_security_group" "container_instance" {
   }
 }
 
-resource "aws_security_group_rule" "container_instance_http_egress" {
-  type        = "egress"
-  from_port   = 80
-  to_port     = 80
-  protocol    = "tcp"
-  cidr_blocks = ["0.0.0.0/0"]
-
-  security_group_id = "${aws_security_group.container_instance.id}"
-}
-
-resource "aws_security_group_rule" "container_instance_https_egress" {
-  type        = "egress"
-  from_port   = 443
-  to_port     = 443
-  protocol    = "tcp"
-  cidr_blocks = ["0.0.0.0/0"]
-
+resource "aws_security_group_rule" "container_instance_all_http_ingress" {
+  type              = "ingress"
+  from_port         = 80
+  to_port           = 80
+  protocol          = "tcp"
   security_group_id = "${aws_security_group.container_instance.id}"
+  cidr_blocks       = ["${var.vpc_external_access_cidr_block}"]
 }
 
 resource "aws_security_group_rule" "container_instance_bastion_ssh_ingress" {
-  type      = "ingress"
-  from_port = 22
-  to_port   = 22
-  protocol  = "tcp"
-
+  type                     = "ingress"
+  from_port                = 22
+  to_port                  = 22
+  protocol                 = "tcp"
   security_group_id        = "${aws_security_group.container_instance.id}"
   source_security_group_id = "${module.vpc.bastion_security_group_id}"
 }
 
-resource "aws_security_group_rule" "container_instance_alb_api_server_all_ingress" {
-  type      = "ingress"
-  from_port = 0
-  to_port   = 65535
-  protocol  = "tcp"
-
+resource "aws_security_group_rule" "container_instance_alb_all_ingress" {
+  type                     = "ingress"
+  from_port                = 0
+  to_port                  = 65535
+  protocol                 = "tcp"
   security_group_id        = "${aws_security_group.container_instance.id}"
   source_security_group_id = "${aws_security_group.server_app_alb.id}"
 }
 
-resource "aws_security_group_rule" "container_instance_alb_api_server_all_egress" {
-  type      = "egress"
-  from_port = 0
-  to_port   = 65535
-  protocol  = "tcp"
-
-  security_group_id        = "${aws_security_group.container_instance.id}"
-  source_security_group_id = "${aws_security_group.server_app_alb.id}"
+resource "aws_security_group_rule" "container_instance_all_egress" {
+  type              = "egress"
+  from_port         = 0
+  to_port           = 0
```

Comment:
I don't think so; the original security group had a rule to allow all egress traffic from the container instance.